uber/templates/signups/shifts.html

```diff
@@ -116,21 +111,24 @@
             header: {
                 left: 'prev,next today',
                 center: 'title',
-                right: 'agendaDay,listWeek'
-            },
-			buttonText : {
-                today : 'Today',
-                agendaWeek : 'Week',
-                agendaDay : 'Day',
-                listWeek : 'List'
+                right: 'agendaDay,agendaConWeek,listConDuration'
             },
 			views: {
                 agendaDay: { buttonText: 'Day' },
-                listWeek: { buttonText: 'Week' }
+                agendaConWeek: {
+                    type: 'agenda',
+                    duration: { days: {{ cal_length }} },
+                    buttonText: 'Calendar'
+                },
+                listConDuration: {
+                    type: 'list',
+                    duration: { days: {{ cal_length }} },
+                    buttonText: 'List'
+                }
             },
-            defaultView: 'listWeeks',
-            editable: true,
-            slotDuration: '00:15:00',
+            defaultView: 'listConDuration',
+            defaultDate: '{{ start_day.strftime('%Y-%m-%d') }}',
+            slotDuration: '00:30:00',
```

Comment:
I changed the slotDuration to 30 minutes instead of 15 -- even though we have an 'extra 15' option, it doesn't actually factor into a job's end time anyway.